app/views/tag/show/_tab_content.html.erb

```diff
@@ -66,6 +66,21 @@
         <%= render partial: "tag/contributors" %>
       </div>
   <% end %>
-  <br />
+  <br>
 
 </div>
+
```

Comment:
Why is this change required? Was this not happening via CSS?

Reply:
It wasn't working via CSS only, hence the bug. I've not been able to figure out why yet, but it works fine now with the jquery code I included.

Reply:
This file was modified to solve a different issue, not this one. But the code wasn't working with just the css. so I had to include the jquery code.